cli/multisig.go

```diff
@@ -399,7 +403,7 @@ var msigProposeCmd = &cli.Command{
 		},
 	},
 	Action: func(cctx *cli.Context) error {
-		if cctx.NArg() < 3 {
+		if cctx.NArg() != 3 {
```

Comment:
check the block directly below this, this command has a bit of nuance, let's just leave it alone

Reply:
should I replace the `showHelp` with `IncorrectNumArgsWithHint` the one we wrote above

Reply:
no, this one's got too much nuance, leave multisig.go alone, it's got fairly sophisticated handling already